netlogo-gui/src/test/app/infotab/InfoFormatterTests.scala

```diff
@@ -104,13 +105,13 @@ class InfoFormatterTests extends FunSuite {
 
   // in contrast, here no <pre> because no blank line
   test("fenced code blocks 2") {
-    assertResult("""|<p>foo<br/><code>
-              |   bar
-              |  baz
-              |qux
-              |</code><br/>oof</p>""".stripMargin) {
+    assertResult("""|<p>foo<br/><code class="text">
```

Comment:
Thinking about it again, I think I know what actually happens here.

Because it is part of a paragraph (there's no blank line), there is *no special meaning to ```*. The first two backticks contain nothing so they render as nothing, then there is another pair of backticks that contain the "block" (inlined though), and then there is another empty inlined code.

This means that:
 - This actually works correctly because we parse inline code as NetLogo code (that should be colorized)
 - This test should be renamed